tests/test_http_client.py

```
@@ -0,0 +1,845 @@
+    success) so ``request()`` returns; otherwise the count is the number
+    of attempts before raising.
+    """
+    captured = _stub_httpx_client(client, list(responses))
+    try:
+        client.request(
+            responses[0].request.method if responses[0].request else "GET",
+            "/any",
+            json={},
+        )
+    except Exception:
+        pass
```

Ruff (0.16.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)